src/System.Text.Json/src/Resources/Strings.resx

```diff
@@ -362,6 +362,9 @@
   <data name="UnexpectedEndOfDataWhileReadingComment" xml:space="preserve">
     <value>Unexpected end of data while reading a comment.</value>
   </data>
+  <data name="CannotSkip" xml:space="preserve">
+    <value>Cannot skip tokens on partial JSON. Either get the whole payload and set _isFinalBlock to true or call TrySkip.</value>
```

Comment:
Why is there an underscore in the resource string?  The caller can't set the field.   You either mean the parameter (no _) or the property (no _, capital letter)